internal/rpchelp/helpdescs_en_US.go

```diff
@@ -685,16 +687,20 @@ var helpDescsEnUS = map[string]string{
 	"walletislocked--result0":  "Whether the wallet is locked",
 
 	// WalletInfoCmd help.
-	"walletinfo--synopsis":              "Returns global information about the wallet",
-	"walletinforesult-daemonconnected":  "Whether or not the wallet is currently connected to the daemon RPC",
-	"walletinforesult-unlocked":         "Whether or not the wallet is unlocked",
-	"walletinforesult-cointype":         "Active coin type. Not available for watching-only wallets.",
-	"walletinforesult-txfee":            "Transaction fee per kB of the serialized tx size in coins",
-	"walletinforesult-ticketpurchasing": "Whether or not the wallet is currently purchasing tickets",
-	"walletinforesult-votebits":         "Vote bits setting",
-	"walletinforesult-votebitsextended": "Extended vote bits setting",
-	"walletinforesult-voteversion":      "Version of votes that will be generated",
-	"walletinforesult-voting":           "Whether or not the wallet is currently voting tickets",
+	"walletinfo--synopsis":                    "Returns global information about the wallet",
+	"walletinforesult-daemonconnected":        "Whether or not the wallet is currently connected to the daemon RPC",
+	"walletinforesult-unlocked":               "Whether or not the wallet is unlocked",
+	"walletinforesult-cointype":               "Active coin type. Not available for watching-only wallets.",
+	"walletinforesult-txfee":                  "Transaction fee per kB of the serialized tx size in coins",
+	"walletinforesult-ticketpurchasing":       "Whether or not the wallet is currently purchasing tickets",
+	"walletinforesult-defaultvotebits":        "Default vote bits setting",
+	"walletinforesult-ticketsvotebits":        "List of saved per-ticket vote bits",
```

Comment:
With the change to not cache per-ticket votebits, I'll be removing this extra data returned with the `walletinfo` json-rpc cmd response. While I think it is relevant data to return, it'll require reading the per-ticket votebits from the db each time the `walletinfo` request is received. I can make that happen but want @jrick's take on it first.

Reply:
that's fine, walletinfo is overloaded anyways (and wrong in some ways, see ticketpurchasing for example).